src/Common/src/Interop/Windows/HttpApi/Interop.HttpApi.cs

```diff
@@ -2,14 +2,14 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.Win32.SafeHandles;
 using System;
 using System.Collections.Generic;
 using System.Net;
 using System.Net.Sockets;
 using System.Runtime.InteropServices;
 using System.Security.Authentication.ExtendedProtection;
 using System.Threading;
+using Microsoft.Win32.SafeHandles;
```

Comment:
Why is this line moving?  We normally keep things in alphabetical order.

Reply:
Oh, that's default behavior of VS Power Tools. I can change that back.

Reply:
For the curious, the rule being observed here is "`System` first", which may be something we should observe.